src/python/WMCore/Storage/StageOutMgr.py

```diff
@@ -176,6 +176,21 @@ def initialiseOverride(self):
         self.fallbacks.append(overrideParams)
         return
 
+# MOVE to dashboardAPI when #6420 pull request is merged
```

Comment:
Seems to me this should have a more specific name if you want to move it to Dashboard API